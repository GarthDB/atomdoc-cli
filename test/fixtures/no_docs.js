function formatComparison(label, basicComparison) {
  const style = (basicComparison.valid) ? complete : missing;
  return [label, style(basicComparison.inspectorValue), style(basicComparison.atomDocValue)];
}
