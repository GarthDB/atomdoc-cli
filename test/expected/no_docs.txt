./test/fixtures/no_docs.js formatComparison line 1
Function on line 1 is missing documentation.